app/components/UserList/UserList.tsx

```diff
@@ -17,18 +57,40 @@ const UserList = () => {
 
   return (
     <ul className="list-group">
+      {jitsiConferenceJoined && localConferenceUser && (
+        <li className={`text-center ${classes.buttons}`}>
+          <div className="btn-group text-center">
+            <ConferenceSwitchButton
+              miniRoom="MAIN"
+              currentMiniRoom={localConferenceUser.miniRoom}
+            />
+            <ConferenceSwitchButton
+              miniRoom="ORI"
+              currentMiniRoom={localConferenceUser.miniRoom}
+            />
+            <ConferenceSwitchButton
+              miniRoom="MOTI"
+              currentMiniRoom={localConferenceUser.miniRoom}
+            />
```

Comment:
```suggestion
{
    _.map(CONFERENCE_ROOMS, room => (
           <ConferenceSwitchButton
              key={room}
              miniRoom={room}
              currentMiniRoom={localConferenceUser.miniRoom}
            />
    ))
}
```

Define `CONFERENCE_ROOMS` as array in conferences module and import it here

```
export const CONFERENCE_ROOMS = ['MAIN', 'ORI', 'MOTI']
```